docs/infrastructure/deployment-targets/cloud-target-discovery/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Cloud Target Discovery
+description: Cloud resources can be discovered and registered as deployment targets by Octopus
+position: 100
+---
+
+:::hint
+The Early Access Preview of the cloud target discovery feature was added in **Octopus 2022.1** and is enabled via **{{Configuration, Features}}**. The preview does not currently include removal of targets when the cloud resource is removed, this is on our roadmap.
+:::
+
+Octopus can discover deployment targets during deployments using tags added to your cloud resources. Target discovery takes place during deployment, and is useful when your deployment process creates your target cloud infrastructure before deploying software to it.
+
+:::hint
+We recommend cloud target discovery over the existing [dynamic infrastructure](/docs/infrastructure/deployment-targets/dynamic-infrastructure/index.md) functionality.
+:::
+
+To enable discovery use the following steps.
+
+## Configure credentials for discovery
+
+To perform discovery of targets from cloud resources, Octopus uses a well-known set of variables configured within your project to provide the credentials for authentication to the cloud provider. Variables can be [scoped](/docs/projects/variables/index.md#scoping-variables) allowing for different credentials to be used when performing discovery as required (for example in different environments).
+
+The variables used are different for each supported cloud provider.
+
+### Azure
+
+To discover Azure cloud resources, Octopus uses an [Azure account](/docs/projects/variables/azure-account-variables.md) variable named **Octopus.Azure.Account**.
```

Comment:
I wonder if this section either needs to labour the point of the name in stronger wording, or a hint to say something like:

> If you have another Azure account variable named differently, Octopus won't look for Azure cloud resources to discover. The account must be named **Octopus.Azure.Account**.

My reasoning is I can imagine a scenario where a customer doesn't see the importance of the name of the variable, and instead just *assumes* it would work if they have any Azure account variable.

Reply:
Yep it's prob good to be extra specific here, I might change it to:

```suggestion
To discover Azure cloud resources, Octopus will look for a specifically named [Azure account](/docs/projects/variables/azure-account-variables.md) variable **Octopus.Azure.Account**.
```

Reply:
Yes, I like that much more than my hint 😄